dynamic-data-masking-redshift/README.md

```diff
@@ -0,0 +1 @@
+# dynamic-data-masking
```

Comment:
@dave-connors-3 — let's put a link to the Discourse article here when it's published :)

Reply:
agreed!